aiida/manage/external/rmq.py

```diff
@@ -168,7 +169,14 @@ def _continue(self, communicator, pid, nowait, tag=None):
 
             raise gen.Return(future.result())
 
-        result = yield super(ProcessLauncher, self)._continue(communicator, pid, nowait, tag)
+        try:
+            result = yield super(ProcessLauncher, self)._continue(communicator, pid, nowait, tag)
+        except ValueError as exception:
```

Comment:
One more question: You are only catching `ValueError` here.
What if other exceptions happen in the call above?
Shouldn't the process node always be taken care of?

Reply:
You are right, I had to update the logic and extended the comment with explanation